fluss-lake/fluss-lake-paimon/src/main/java/com/alibaba/fluss/lake/paimon/PaimonLakeCatalog.java

```diff
@@ -97,11 +98,11 @@ public void createTable(TablePath tablePath, TableDescriptor tableDescriptor)
         }
     }
 
-    private void createTable(Identifier tablePath, Schema schema)
+    private void createTable(Identifier tablePath, Schema schema, boolean ifNotExists)
             throws Catalog.DatabaseNotExistException {
         try {
             // not ignore if table exists
-            paimonCatalog.createTable(tablePath, schema, false);
+            paimonCatalog.createTable(tablePath, schema, ifNotExists);
```

Comment:
if we support  `ifNotExists`, how can we ensure the data offset of exists lake table can be consistent with fluss table's offset?

Reply:
See the discussion in [here](https://github.com/apache/fluss/issues/1260#issuecomment-3034775576), so, let's pending this pr.